vxquery-core/src/main/java/org/apache/vxquery/datamodel/values/XDMConstants.java

```diff
@@ -19,11 +19,11 @@
 import java.io.IOException;
 import java.util.Arrays;
 
-import org.apache.vxquery.datamodel.builders.sequence.SequenceBuilder;
-
 import org.apache.hyracks.data.std.api.IPointable;
 import org.apache.hyracks.data.std.primitive.BooleanPointable;
 import org.apache.hyracks.data.std.util.ArrayBackedValueStorage;
+import org.apache.vxquery.datamodel.builders.jsonItem.ObjectBuilder;
+import org.apache.vxquery.datamodel.builders.sequence.SequenceBuilder;
 
 public class XDMConstants {
     private static final byte[] BOOLEAN_TRUE_CONSTANT;
```

Comment:
What should be the valueTag for NULL? I could return that if a value is not found for a given key.